src/js/node/util.ts

```diff
@@ -280,6 +285,13 @@ function styleText(format, text) {
   return `\u001b[${formatCodes[0]}m${text}\u001b[${formatCodes[1]}m`;
 }
 
+function getSystemErrorName(err: any) {
```

Comment:
Can't we do this in native code and validate the arguments there?

Reply:
this is public api https://nodejs.org/docs/latest/api/util.html#utilgetsystemerrornameerr